bit_manipulation/decimal_to_binary.py

```diff
@@ -0,0 +1,43 @@
+def decimal_to_binary(number: int) -> str:
+    """
+    Return a binary string
+    >>> decimal_to_binary(0)
+    '0'
+    >>> decimal_to_binary(1)
+    '1'
+    >>> decimal_to_binary(8)
+    '1000'
+    >>> decimal_to_binary(110)
+    '1101110'
+    >>> decimal_to_binary(-20)
+    '-10100'
+    >>> decimal_to_binary("abc")
+    Traceback (most recent call last):
+        ...
+    ValueError: Input is not valid
+    >>> decimal_to_binary([])
+    Traceback (most recent call last):
+        ...
+    ValueError: Input is not valid
+    """
+    if not isinstance(number, int):
+        raise ValueError("Input is not valid")
+    if number == 0:
+        return "0"
+    flag = 0
+    if number < 0:
+        number *= -1
+        flag = 1
+    answer = ""
```

Comment:
Please write code that can actually convert decimal to binary, your code just converts int to binary. Even the build-in function `bin` can do that.

I have written a sample code to do so since I only have view permission, I want you to make some amendments.

```Python
a = ""
r = ""
count = 0
d = number - int(number)
number = int(number)
while(d and count<10):
     d = d*2
     r+=str(int(d))
     if int(d)==d:
            break
     else:
            d = d -int(d)
            count+=1
while number:
     a = str(number%2) + a
     number = number >> 1
a = a+"."+r
print(a)
```